dashboard/app/models/user_level.rb

```diff
@@ -23,6 +23,10 @@ class UserLevel < ActiveRecord::Base
   belongs_to :level
   belongs_to :script
 
+  def submitted?
```

Comment:
It would be helpful to expand the class-level comment above to mention submitted attempts as well